.gitignore

```
@@ -0,0 +1,32 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

```suggestion
node_modules
```

removing the leading slash allows any node modules directory to be gitignored (note that the way this is now, the example/node_modules directory doesn't get ignored